ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/dcatserver/repositories/DcatServerRepository.java

```diff
@@ -13,4 +13,6 @@ public interface DcatServerRepository {
 	DcatServer saveServerDcat(DcatServer dcatServer);
 
 	void deleteServerDcat(String id);
+
+	Optional<DcatServer> findSingleDcatServer();
```

Comment:
is there a need to say "single" ? I think we use find[entity]() for most of our calls to get one item?

Reply:
I did this because we do not provide an identifier. Usually you expect a single result when providing an identifier or a list when providing nothing. However when you expect just one result, it is common (in hibernate https://docs.jboss.org/hibernate/orm/4.3/javadocs/org/hibernate/result/ResultSetOutput.html#getSingleResult()) to explicitly mention this.

However I feel like having an id for the dcatServer is actually very user unfriendly (there is no way to know the id after creating, so all the other changes become impossible without consulting the db manually for the id) and I plan to propose on the demo prep to change this. When we work without an id, this single will become useless as well.